teachopencadd/talktorials/T036_e3_equivariant_gnn/talktorial.ipynb

```diff
@@ -5,13 +5,13 @@
    "cell_type": "markdown",
```

Comment:
<blockquote><strong>Talktorial 33</strong></blockquote><p>The notation is:</p><p><strong>Talktorial T033</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>